src/reducers/filter.js

```diff
@@ -0,0 +1,10 @@
+import { CHANGE_FILTER } from '../actions';
+
+export default (state = '', action) => {
```

Comment:
- [ ] The state should default to `All` not an empty string [screenshot from Pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone4/ItDefaultsToAll.png).